api/STRUCTURE.md

```diff
@@ -0,0 +1,83 @@
+# All of Us Workbench API Structure & Technologies
+
+The _All of Us_ Workbench API is largely built on [Spring Boot](https://spring.io/projects/spring-boot). It also uses [Swagger](https://swagger.io/) for autogenerating REST APIs.
+
+## Database
+
+The database is a [Google Cloud SQL](https://cloud.google.com/sql/) instance running MySQL. We modify its schema using [Liquibase](https://www.liquibase.org/) migrations, which live in [db/changelog/](https://github.com/all-of-us/workbench/tree/master/api/db/changelog).
```

Comment:
I'd probably say "The primary application database is..." so as not to hurt BigQuery's feelings.